testsuites/syncgateway/functional/tests/test_cbs_collections.py

```diff
@@ -506,7 +504,169 @@ def test_import_filters(scopes_collections_tests_fixture):
+
+    sg_client, sg_admin_url, db, scope, collection = scopes_collections_tests_fixture
+    random_suffix = str(uuid.uuid4())[:8]
+
+    # 1. Verify that global stats and scopes/collection stats can be procured
+    collection_stats_keys = ["num_doc_writes", "num_doc_reads", "doc_writes_bytes", "doc_reads_bytes", "import_count", "sync_function_time",
+                             "sync_function_count", "sync_function_reject_access_count", "sync_function_reject_count", "sync_function_exception_count"]
+
+    stats = sg_client.get_expvars(sg_admin_url)
+
+    try:
+        global_stats = stats["syncgateway"]["global"]
+    except KeyError:
+        assert False, "Could not retrieve global stats via _expvar endpoint"
+    else:
+        assert len(global_stats) != 0, f"Global stats are not reported correctly \n + {str(global_stats)}"
+
+    collection_stats = try_get_collection_stats(db, scope, collection, stats)
+    assert isinstance(collection_stats, dict), "Could not retrieve collection stats via _expvar endpoint"
+
+    for key in collection_stats_keys:
+        try:
+            assert collection_stats[key] == 0, f"{key} statistic should be 0, got {collection_stats[key]}"
+        except KeyError:
+            assert False, f"{key} statistic does not exist"
+
+    # 2. Add two new collections on CB server, rename SGW collection to map to it, adding new collection and enable import + sync functions
+    second_collection = "second_collection" + random_suffix
+    third_collection = "third_collection" + random_suffix
+
+    second_collection_doc_key = "doc2" + random_suffix
+    third_collection_doc_key = "doc3" + random_suffix
+
+    cb_server.create_collection(bucket, scope, second_collection)
+    cb_server.create_collection(bucket, scope, third_collection)
+
+    custom_sync = f"function(doc, oldDoc) {{requireAccess(\"{third_collection}\"); if (doc._id == \"forbidden_doc\") {{throw({{incrementExceptionStat: \"forbidden doc to increment exception stat in test\"}})}} channel(\"{third_collection}\")}}"
+
+    db_config = {"bucket": bucket, "scopes": {scope: {"collections": {third_collection: {"sync": custom_sync}, second_collection: {"sync": simple_sync_function(second_collection)}}}},
+                 "num_index_replicas": 0, "import_docs": True, "enable_shared_bucket_access": True}
+    admin_client.post_db_config(db, db_config)
+    admin_client.wait_for_db_online(db, 60)
+
+    # 3. Verify that stats parameters update to reflect new collections
+    renamed_stats = sg_client.get_expvars(sg_admin_url)
+
+    second_collection_stats = try_get_collection_stats(db, scope, second_collection, renamed_stats)
+    third_collection_stats = try_get_collection_stats(db, scope, third_collection, renamed_stats)
+    assert isinstance(second_collection_stats, dict), f"syncgateway.per_db.{db}.per_collection.{scope}.{second_collection} was not found in stats after collection added to SGW database"
```

Comment:
Maybe it's worth getting that check inside try_get_collection_stats? It's done in several places, so it could save a few lines, and make the test slightly easier to read

Reply:
Sounds reasonable, will add change